### ISSUE TITLE
Compiler crash(?) with modified plugin myinfo values

### ISSUE THREAD

**Issue by Bara** [alliedmodders/sourcepawn#466]
This was reported on discord by Wend4r, I just create this issue before we forget this.

Tested and effected compilers: 1.10.6462, 1.10.6478, 1.11.6595 and 1.11.6522

Code that's work:
```sp
public Plugin myinfo =
{
    name = "name", 
    author = "author", 
    description = "description", 
    version = "version", 
    url = "url"
};
```

But if we change the default names to something else like `name` to `testName`, then the compile process will be terminated and return the exit code 1 (`The terminal process terminated with exit code: 1`). Example Code:
```sp
public Plugin myinfo =
{
    testName = "name", 
    author = "author", 
    description = "description", 
    version = "version", 
    url = "url"
};
```

**Comment by Fyren** [comment]
Seems to only happen on 1.10, not 1.9 or master.  Should be fixed in SM whenever the pinned SP repo is updated.

**Comment by Bara** [comment]
Wouldn't it be better to keep it open, since it actual reproducible with the [current version of SourcePawn in SourceMod](https://github.com/alliedmodders/sourcepawn/tree/305b59efbf97875da0ebfd46038fdfe519dbe077)?

**Comment by dvander** [comment]
Unless it solves a critical bug or is needed to grease the wheels for 1.11 adoption, I'm trying not to backport fixes. It's a large burden.

**Comment by Fyren** [comment]
I closed this because I saw the SM 1.10 merge but forgot Bara reported the issue happens in whatever version of SP is pinned in SM 1.11.  (I don't know if keeping issues open till SM changes pinned revisions is helpful, personally.)

This wasn't a backport, just a dumb mistake fix.  On master, I believe dvander rewrote this in the new parser, so the bug isn't present there.
